backend/core/src/app.module.ts

```diff
@@ -1,3 +1,12 @@
+// dotenv is a dev dependency, so conditionally import it (don't need it in Prod).
```

Comment:
Doesn't making something a dev dependency exclude it from prod builds? I don't think I've ever seen something conditionally imported like this